src/qml/qml.qrc

```diff
@@ -42,5 +42,7 @@
         <file>GeometryRenderer.qml</file>
         <file>GeometryHighlighter.qml</file>
         <file>FeatureListSelectionHighlight.qml</file>
+        <file>editorwidgets/ValueRelation.qml</file>
+        <file>editorwidgets/RelationWidget.qml</file>
```

Comment:
I would put that file into a different folder.
Files in this folders should all have names that exactly match the widget ids.